test/unit/bin/helpers/capabilityHelper.js

```diff
@@ -249,6 +249,136 @@ describe("capabilityHelper.js", () => {
           chai.assert.fail("Promise error");
         });
     });
+
+    context("specs and env from run_setting", () => {
+      it("sets specs list is present", () => {
```

Comment:
no test where specs list is an Array or passed args is of Array type

Reply:
As call to `util.setUserSpecs` and `util.setTestEnvs`are [made in runs.js](https://github.com/browserstack/browserstack-cypress-cli/pull/46/files/961eb70b7347802f7cad51a6192c92b6ec21df39#diff-94102bcdbd99f5dee9c6696fdae0a977R27-R32) and then final state(String) is passed into capabilityHelper. Hence, capabilityHelper always receives string.  

specs for array conditions are covered in the util.js itself: https://github.com/browserstack/browserstack-cypress-cli/pull/46/files/961eb70b7347802f7cad51a6192c92b6ec21df39#diff-175b2b2ef3763bf0f62fe44137aa5d34R350